src/components/dialog/index.tsx

```diff
@@ -9,33 +9,26 @@ export default function Dialog({
   onDismiss,
   label,
 }: DialogProps) {
-  const AnimatedDialogOverlay = m(DialogOverlay)
+  const shouldReduceMotion = useReducedMotion()
+  const AnimatedDialogOverlay = motion(DialogOverlay)
```

Comment:
just noticed this definition is inside the component body, can we move it out? Otherwise it recreates a new component on every render.

Reply:
Good call! Updating

Reply:
https://github.com/hashicorp/dev-portal/pull/626/commits/27c2dd1fbeb64f5b7498577dca4f2f38e5d03e51 ✅